freestyle-cache/shared/src/test/scala/TestContext.scala

```diff
@@ -29,7 +30,7 @@ trait CacheTestContext extends BeforeAndAfterEach { self: Suite =>
   private[this] implicit val rawMap: KeyValueMap[Id, String, Int] =
     new ConcurrentHashMapWrapper[Id, String, Int]
 
-  private[this] implicit val idInt: Id ~> Id = FunctionK.id[Id]
+  private[this] implicit val idInt: FSHandler[Id, Id] = FunctionK.id[Id]
```

Comment:
Just a very small minor thing, but can we rename `idInt` to `idHandler while we are changing things here?
(I find using `Int` as a shorthand for interpreter a bit confusing)